test/IDE/complete_decl_attribute_feature_requirement.swift

```diff
@@ -0,0 +1,154 @@
+// RUN: %target-swift-ide-test -code-completion -source-filename %s -code-completion-token=KEYWORD2 | %FileCheck %s -check-prefixes=KEYWORD2,KEYWORD2_DISABLED
```

Comment:
Note that you can use `%batch-code-completion` here instead. It has the benefit of not needing the many RUN lines + can re-use the ast context. Thanks for adding these by the way, it's very much appreciated!

Reply:
I had to modify the tool with a feature that could add suffixes to the check prefixes, but this definitely cleaned things up quite a bit!

Reply:
Oh nice, thank you! CC @rintaro